app/com/linkedin/drelephant/mapreduce/fetchers/MapReduceFSFetcherHadoop2.java

```diff
@@ -209,6 +209,7 @@ public MapReduceApplicationData fetchData(AnalyticJob job) throws IOException {
     String histFile = files.getJobHistPath();
     String appId = job.getAppId();
     String jobId = Utils.getJobIdFromApplicationId(appId);
+    logger.info(" AppID "+appId);
```

Comment:
Looks like these logs haven added while testing. Please verify and remove the unwanted or change them to debug.